ui/article-renderer/src/main/res/layout/list_item_article.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
you should use databinding for this layout. see: https://github.com/CruGlobal/godtools-android/blob/develop/ui/article-renderer/src/main/res/layout/list_item_category.xml